### DISCUSSION TITLE
How much energy does the Raspberry Pi consume in a day?

### DISCUSSION THREAD

**Discussion by rern** [rern/rAudio#116] [Tips]
[How much energy does the Raspberry Pi consume in a day?](https://raspberrypi.stackexchange.com/questions/5033/how-much-energy-does-the-raspberry-pi-consume-in-a-day)

**Comment by findnix** [comment]
RPi4 8GB: Measured about 1 A idle (including a m.2 500 GB SSD and a Pro-Ject Pre Box S2 Digital DAC, sourced via RasbPi USB) ) and about 1,4 A when playing a DSD file.
mpd is the only CPU consuming process with about 50% of one of 4 cores.
Avg 1,2 A ->  1,2A*5V*24h = 144 Wh 
Which is close to this table: https://raspi.tv/2019/how-much-power-does-the-pi4b-use-power-measurements 
I would say in common between 100 Wh and 200 Wh.
